### PR TITLE
Closes #446 | Add/Update Dataloader voxlingua

### LINKED ISSUE #446: Create dataset loader for VoxLingua107
Dataloader name: `voxlingua/voxlingua.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?voxlingua

|     Dataset| voxlingua |
|-------------|---|
| Description | VoxLingua107 is a comprehensive speech dataset designed for training spoken language identification models. It comprises short speech segments sourced from YouTube videos, labeled based on the language indicated in the video title and description. The dataset covers 107 languages and contains a total of 6628 hours of speech data, averaging 62 hours per language. However, the actual amount of data per language varies significantly. Additionally, there is a separate development set consisting of 1609 speech segments from 33 languages, validated by at least two volunteers to ensure the accuracy of language representation. |
| Subsets | - |
| Languages | ceb, ind, jav, zlm, mya, sun, tha, tgl, vie, war, khm, lao |
| Tasks | Spoken Language Identification |
| License | Creative Commons Attribution 4.0 (cc-by-4.0) |
| Homepage | https://bark.phon.ioc.ee/voxlingua107/ |
| HF URL | https://huggingface.co/TalTechNLP/voxlingua107-epaca-tdnn |
| Paper URL | https://arxiv.org/abs/2011.12998 |

### PR DESCRIPTION
**Pull request by sabilmakbar**
Please name your PR title and the first line of PR message after the issue it will close. You can use the following examples:

**Title**: Closes #446 | Add/Update Dataloader voxlingua

**First line PR Message**: Closes #446 

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/{my_dataset}/{my_dataset}.py` (please use only lowercase and underscore for dataset folder naming, as mentioned in dataset issue) and its `__init__.py` within `{my_dataset}` folder.
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_LOCAL`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py` or `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py --subset_id {subset_name_without_source_or_seacrowd_suffix}`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

For 6 & 7, I have tested it using `ceb` language only since the data folder is quite large (up to 6GB per each language)

### CODE REVIEW

File: seacrowd/sea_datasets/voxlingua/voxlingua.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+                version=datasets.Version(version),
+                description=f"{_DATASETNAME} {config_name_prefix} schema for {task_obj.name}",
+                schema=f"{config_name_prefix}_{config_name_suffix}",
+                subset_id=f"{config_name_prefix}_{config_name_suffix}",
+            )
+        ]
+    return config_list
+
+
+class VoxLingua(datasets.GeneratorBasedBuilder):
```
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/voxlingua/voxlingua.py:109]:
```suggestion
class VoxLinguaDataset(datasets.GeneratorBasedBuilder):
```

**Review by ljvmiranda921 — COMMENTED** [review]
LGTM! Just have a small comment. Thank you!

**Comment by sabilmakbar** [comment]
> LGTM! Just have a small comment. Thank you!

Thanks LJ, comment addressed, pls have a check, thanks!

**Review by MJonibek — APPROVED** [review]
Checked, LGTM

**Review by ljvmiranda921 — APPROVED** [review]
LGTM! Merging in a few minutes

**Pull request merged**